src/Microsoft.AspNetCore.Server.Kestrel.Core/CoreStrings.resx

```diff
@@ -0,0 +1,123 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
So we can assert the exception being thrown has the message we expect. Per this comment: https://github.com/aspnet/KestrelHttpServer/pull/1706#discussion_r111872235